examples/todo/src/openapi-spec.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {TodoListApplication} from './application';
+
+/**
+ * Export the OpenAPI spec from the application
+ */
+async function exportOpenApiSpec(): Promise<void> {
+  const config: ApplicationConfig = {
+    rest: {
+      port: +(process.env.PORT ?? 3000),
+      host: process.env.HOST ?? 'localhost',
+    },
+  };
+  const outFile = process.argv[2] ?? '';
+  const app = new TodoListApplication(config);
+  await app.boot();
+  await app.exportOpenApiSpec(outFile);
+}
+
+if (require.main === module) {
```

Comment:
What's the purpose of this check? When we don't all `exportOpenApiSpec` then this module is a no-op. Can we remove it please, also for consistency with `src/migrate.ts`?